### PR TITLE
Bugfix 311 - Fix ipv4 formatting issue

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### SUMMARY

fixes: https://github.com/ansible-collections/cisco.iosxr/issues/311

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/fragments/bugix_l3_interface.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```
Comment (KB-perByte) [inline changelogs/fragments/bugix_l3_interface.yaml:2]: This should be a major release I suppose!

Comment (ashwini-mhatre) [reply changelogs/fragments/bugix_l3_interface.yaml]: Ack

File: changelogs/fragments/bugix_l3_interface.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
+  - Fix issue in ipv4 address formatting(l3-interface).
```
Comment (Qalthos) [inline changelogs/fragments/bugix_l3_interface.yaml:3]:
```suggestion
  - iosxr_l3_interfaces - fix issue in ipv4 address formatting. (https://github.com/ansible-collections/cisco.iosxr/issues/311)
```

**Review by Qalthos — CHANGES_REQUESTED** [review]

**Comment by KB-perByte** [comment]
@ashwini-mhatre Things look good to me! +1 but are we planning for a major release for IOSXR?

**Comment by ashwini-mhatre** [comment]
> @ashwini-mhatre Things look good to me! +1 but are we planning for a major release for IOSXR?

yes as we are planning netcommon major release we are also planning major release for xr

**Review by Qalthos — APPROVED** [review]

**Pull request merged**